jobs/openshift4/cluster/integreatly/install/Jenkinsfile

```diff
@@ -0,0 +1,94 @@
+#!groovy
+@Library('delorean-pipeline-library')
+
+String cluserAPI = "https://api.${CLUSTER_DOMAIN}:6443"
+Boolean cleanup = params.CLEANUP
+
+
+node('cirhos_rhel7') {
+     timeout(60) { ansiColor('gnome-terminal') { timestamps {
+    //currentBuild.displayName = "${currentBuild.displayName} ${CLUSTER_DOMAIN}"
+    currentBuild.description = "cluster: ${CLUSTER_DOMAIN}\n"
+
+
+    stage('Login to OpenShift 4 cluster') {
+        sh "oc login ${cluserAPI} --insecure-skip-tls-verify=true -u ${ADMIN_USERNAME} -p ${ADMIN_PASSWORD}"
+    } 
+
+    stage('Clone Integreatly Operator') {
+        dir('integreatly-operator') {
+            checkout([
+                $class: 'GitSCM',
+                branches: [[name: "${INTEGREATLY_OPERATOR_BRANCH}"]],
+                doGenerateSubmoduleConfigurations: false,
+                extensions: [],
+                userRemoteConfigs: [[url: "${INTEGREATLY_OPERATOR_REPOSITORY}"]]
+            ])
+            sh "git clean -xdf"
```

Comment:
What does this do?

Reply:
@mikenairn cloning intly-operator repo afaik, just copy&pasted it from other pipeline